src/main/resources/validationrules/sscs/sscs-judicial-office-holder-mapping.drl

```diff
@@ -267,6 +267,8 @@ when
    $f:  FeatureFlag(status && flagName == FeatureFlagEnum.SSCS_WA_1_0.getValue())
    $jap: JudicialAccessProfile(appointment == "Tribunal Member Financially Qualified",
                                appointmentType == "Fee Paid",
+                               baseLocationId == "1032",
```

Comment:
There is an additional rule (sscs_tribunal_member_baselocation_ticketcode_joh) also generating "SSCS Tribunal Member Financially Qualified" but for appointment == "Tribunal Member" should it be removed?

Reply:
> This seems to be an extra rule from the old config. Which has been removed by the BA in the updated config, hence have removed. See: e0a3add1c3f95db1f393730b0357af1be7357c70